wrangler-service/src/main/java/io/cdap/wrangler/service/gcs/GCSHandler.java

```diff
@@ -88,6 +89,7 @@
  */
 public class GCSHandler extends AbstractWranglerHandler {
   private static final Logger LOG = LoggerFactory.getLogger(GCSHandler.class);
+  private static final int MAX_SAMPLE_ROWS = 5000;
```

Comment:
if we make this change does this mean users who were able to browse more than 5000 records will see different behavior in case first 5000 records had null values in columns (the column types will be unknown)?

Reply:
It's an unlikely scenario, as they would have to small enough data that it wasn't going OOM. 

But it's a good point, we should have a way to change it. I think I'll change this to a default and look in the runtime arguments for a max value as an override.